store/contentSchemas.json

```diff
@@ -0,0 +1,82 @@
+{
+  "definitions": {
+    "StoreSettings": {
+      "title": "VTEX Store",
+      "type": "object",
+      "properties": {
+        "storeName": {
+          "title": "Store Name",
+          "type": "string"
+        },
+        "requiresAuthorization": {
+          "title": "Enables B2B behavior",
+          "type": "boolean"
+        },
+        "titleTag": {
+          "title": "Default title tag",
+          "$ref": "app:vtex.native-types#/definitions/text"
```

Comment:
This is one of the only two differences between this definition and the `settingsSchema`